src/isilon_hadoop_tools/identities.py

```diff
@@ -412,6 +412,7 @@ def cdp_identities(zone):
             'hue': ('hue', set()),
             'impala': ('impala', {'hive'}),
             'kafka': ('kafka', set()),
+            'keyadmin': ('keyadmin', set()),
```

Comment:
Should this be in the `hadoop` group like HDP?
https://github.com/Isilon/isilon_hadoop_tools/blob/deebd9add0c78444ba99189b2ff933b51ef88e6b/src/isilon_hadoop_tools/identities.py#L492